packages/dd-trace/src/ci-visibility/dynamic-instrumentation/worker/index.js

```diff
@@ -81,7 +105,7 @@ async function addBreakpoint (snapshotId, probe) {
     try {
       lineNumber = await processScriptWithInlineSourceMap({ file, line, sourceMapURL })
     } catch (err) {
-      log.error(err)
+      log.error('Error processing script with inline source map')
```

Comment:
Are you sure you don't want to report the original error as well? Ok if you don't, I just wanted to make sure it wasn't an oversight 😊

```suggestion
      log.error('Error processing script with inline source map', err)
```